### PR TITLE
Update to Dogma v0.3.0.

### PR DESCRIPTION
**Pull request by jmalloc**

### CODE REVIEW

File: config/aggregate_test.go [hunk 1]
```diff
@@ -105,53 +105,53 @@ var _ = Describe("type AggregateConfig", func() {
...
 				func(c dogma.AggregateConfigurer) {
 					c.Name("\t \n")
-					c.AcceptsCommandType(fixtures.MessageA{})
-					c.RecordsEventType(fixtures.MessageE{})
+					c.ConsumesCommandType(fixtures.MessageA{})
+					c.ProducesEventType(fixtures.MessageE{})
 				},
 			),
 			Entry(
 				"when the handler does not configure any accepted command types",
```
Comment (koden-km) [inline config/aggregate_test.go]: Are these supposed to still use the "accept" language, or should they use the "consume" language?

File: config/aggregate_test.go [hunk 2]
```diff
@@ -105,53 +105,53 @@ var _ = Describe("type AggregateConfig", func() {
...
-					c.AcceptsCommandType(fixtures.MessageA{})
-					c.AcceptsCommandType(fixtures.MessageA{})
-					c.RecordsEventType(fixtures.MessageE{})
+					c.ConsumesCommandType(fixtures.MessageA{})
+					c.ConsumesCommandType(fixtures.MessageA{})
+					c.ProducesEventType(fixtures.MessageE{})
 				},
 			),
 			Entry(
 				"when the handler does not configure any recorded events",
```
Comment (koden-km) [inline config/aggregate_test.go]: Are these supposed to still use the "recorded" language, or should they use the "produced" language?

File: config/aggregate_test.go (hunk 1 above)
Comment (jmalloc) [reply config/aggregate_test.go]: Yeah,  this should probably change. I'll update it on `master`, but no need for a new tag just yet.

Comment (jmalloc) [reply config/aggregate_test.go]: Oh wait, wrong repo. Yes, will fix.

**Pull request merged**